packages/client/components/markdown/index.tsx

```diff
@@ -94,6 +96,45 @@ const components = () => ({
   style: Null,
 });
 
+const replyComponents = () => ({
+  unicodeEmoji: RenderUnicodeEmoji,
+  customEmoji: RenderCustomEmoji,
+  mention: RenderMention,
+  spoiler: RenderSpoiler,
+  
+  strong: elements.strong,
+  em: elements.emphasis,
+  code: elements.code,
+  del: elements.strikethrough,
+  
+  p: (props: any) => <>{props.children}</>,
+  h1: Null,
+  h2: Null,
+  h3: Null,
+  h4: Null,
+  h5: Null,
+  h6: Null,
+  pre: Null,
+  li: Null,
+  ul: Null,
+  ol: Null,
+  blockquote: Null,
+  table: Null,
+  th: Null,
+  td: Null,
+  a: Null,
+  img: Null,
+  video: Null,
+  figure: Null,
+  picture: Null,
+  source: Null,
+  audio: Null,
+  script: Null,
+  style: Null,
+  time: Null,
+  timestamp: Null,
```

Comment:
I think the rest of these can probably just pass-through their text content where applicable (e.g. headings or lists) and timestamps are also okay to render.

Reply:
Updated!